flow-typed/axios.js.flow

```diff
@@ -1,61 +1,90 @@
-// flow-typed signature: 8f0440c9765369dfa5106787b5e702bc
-// flow-typed version: 480b2adfe4/axios_v0.11.x/flow_>=v0.23.x
+declare module 'axios' {
```

Comment:
Yeah, I installed the flow-typed definition, which fixed a few of my issues actually. But I found a typo or two in the libdef. Submitted a PR to fix those: https://github.com/flowtype/flow-typed/pull/224